steam-condenser.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |s|
   s.summary     = 'Steam Condenser - A Steam query library'
   s.description = 'A multi-language library for querying the Steam Community, Source, GoldSrc servers and Steam master servers'
 
-  s.required_ruby_version = '>= 1.9.2'
+  s.required_ruby_version = '>= 2.0'
```

Comment:
Not sure if this includes JRuby (which is Ruby 1.9 afaik), let's wait what Travis says.